docs/cas-server-documentation/installation/Configuration-Properties.md

```diff
@@ -2105,6 +2105,12 @@ Multifactor authentication bypass settings for this provider are available [here
 # cas.authn.mfa.yubikey.allowedDevices.uid2=yubikeyPublicId2
 ```
 
+#### YubiKey Registration Records Encryption and Signing
+```properties
+# cas.authn.mfa.yubikey.crypto.enabled=true
```

Comment:
Please remove these settings, and instead, only list the group id and link out to the *Commons* page where crypto settings are listed.

Example:
https://apereo.github.io/cas/development/installation/Configuration-Properties.html#jwt-tickets

>The signing key and the encryption key are both JWKs..... Signing & encryption settings for this feature are available here under the configuration key ...